data_tools/add.py

```diff
@@ -102,9 +99,9 @@
         added += 1
     
     print ("About to batch add: {}".format(added, skipped_existing))
-    fcw.set_messages_content_batch(DATASET_ID, messages_to_write)
+    fcw.set_dataset_messages_content_batch(DATASET_ID, messages_to_write)
     print ("Batch add complete: {}, Skipped: {}".format(added, skipped_existing))
     
     print('Updating Ops Dashboard for dataset: {}'.format(DATASET_ID))
     cp.compute_coding_progress(DATASET_ID, force_recount=True)
-    print('Done')
+    print('Done')
```

Comment:
New line

Reply:
thanks, not sure how that slipped through